src/main/java/net/snowflake/client/core/SecureStorageLinuxManager.java

```diff
@@ -70,7 +70,7 @@ private ObjectNode localCacheToJson() {
   public synchronized SecureStorageStatus setCredential(
       String host, String user, String type, String token) {
     if (Strings.isNullOrEmpty(token)) {
-      logger.info("No token provided", false);
+      logger.debug("No token provided", false);
```

Comment:
no issue with debug level however, isn't it weird that empty token is actually not stored but entire function returns a success and it's not indicated as an error. If that should be an error maybe warn would make more sense

Reply:
true, we can change to warn